index.bs

```diff
@@ -816,62 +868,111 @@ To <dfn>handle funky elements</dfn> on a given |element|, run these steps:
-  1. Let |baseline action| be the result of running the steps of the
-     [=determine the baseline configuration for an element=] algorithm
-     for the element |element|.
-  1. Let |config action| be the result of running the steps of the
-     [=determine the effective configuration for an element=] algorithm
-     for the element |element| and the config |config|.
-  1. Return the [=stricter action=] of |baseline action| and |config action|.
-
-Note: The definition of stricter actions ensures that the built-in baseline
-      configuration cannot be overriden, and therefor forms a hard guarantee
-      for all Sanitizer instances. Likewise for attributes.
+
+  1. Let |tokens| be the result of running the [=split on ascii whitespace=]
+     algorithm on |name|.
+  1. If |tokens| contains one string,
+     and if |element| is in the [=HTML namespace=]
+     and if |element|'s [=Element/local name=] is an
+     [=ASCII case-insensitive=] match for the the one string in |tokens|:
+     Return `true`.
+  1. If |tokens| contains two strings
+     and if the first of these strings is "svg"
+     and if |element| is in the [=SVG namespace=]
+     and if |element|'s [=Element/local name=] is an
+     [=ASCII case-insensitive=] match for the second string in |tokens|:
+     Return `true`.
+  1. If |tokens| contains two strings
+     and if the first of these strings is "math"
+     and if |element| is in the [=MathML namespace=]
+     and if |element|'s [=Element/local name=] is an
+     [=ASCII case-insensitive=] match for the second string in |tokens|:
+     Return `true`.
+  1. Return `false`.
 </div>
 
-<div algorithm="effective attribute configuration">
-To determine a Sanitizer |sanitizer|'s
-<dfn>effective attribute configuration</dfn> for an attribute |attr|
-attached to an element |element|, run these steps:
-  1. Let |config| be |sanitizer|'s [=configuration object=].
-  1. Let |baseline action| be the result of running the steps of the
-     [=determine the baseline configuration for an attribute=] algorithm
-     on the attribute |attr|.
-  1. Let |config action| be the result of running the steps of the
-     [=determine the effective configuration for an attribute=] algorithm
-     on the attribute |attr|, with the element |element| and the
-     config |config|.
-  1. Return the [=stricter action=] of |baseline action| and |config action|.
+<div algorithm="attribute matches an attribute match list">
+To determine whether an <dfn>|attribute| matches an [=attribute match
+list=]</dfn> |list|, run these steps:
+
+  1. If |attribute|'s [=Attr/local name=] does not match the
+     [=attribute match list=] |list|'s
+     [key](https://webidl.spec.whatwg.org/#idl-record) and if the key is
+     not `"*"`: Return `false`.
+  1. Let |element| be the |attribute|'s {{ownerElement}}.
+  1. Let |element name| be |element|'s [=Element/local name=].
+  1. If |element| is in either the [=SVG namespace|SVG=] or
```

Comment:
You could future-proof this a little by pointing to https://html.spec.whatwg.org/#foreign-elements, which would presumably include amazing new namespaces we add in the future.